site/gatsby-site/cypress/integration/wordcounts.js

```diff
@@ -0,0 +1,47 @@
+describe('The Word Counts Page', () => {
+  it('successfully loads', () => {
+    cy.visit('/summaries/wordcounts');
+  });
+
+  it('word count table should exist', () => {
+    cy.visit('/summaries/wordcounts');
+    cy.get('.row .list-group');
+    cy.get('.row .list-group > div');
```

Comment:
I agree with the comment above. Also, this seems to be missing assertions? It is getting the DOM nodes but not doing anything with them:

Something like `cy.get('.row .list-group').should('exist');`

Reply:
https://docs.cypress.io/api/commands/should

Reply:
Yeah, the test will fail, but leaving it like that, it is not clear what you are testing. Because that test will pass no matter how many items there are or even if they are visible or not, as long as there is at least one present on the DOM, but that's a little bit open-ended.

You probably want something like `cy.get('.row .list-group').should('have.length', 2).and('be.visible')` (didn't test it).